apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -93,6 +95,34 @@ export const setValidAssignments = (validCourses, validScripts) => ({
 });
 
 /**
+ * Get our query string. Provided as a method so that tests can mock this.
+ */
+export function getQueryString_() {
+  return window.location.search;
+}
```

Comment:
Don't know if there's a better way to be able to stub this, but this seems to work (semi-copied from what we do in experiments.js)